src/frontend/sharedComponents/Thumbnail/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React, {FC} from 'react';
+import {Dimensions, ScrollView, StyleSheet} from 'react-native';
+
+import {Photo} from '../../contexts/PhotoPromiseContext/types';
+import {Thumbnail} from './Thumbnail';
+import {AudioThumbnail} from './AudioThumbnail';
+// import {useNavigationFromHomeTabs} from '../../hooks/useNavigationWithTypes';
+
+const spacing = 10;
+const minSize = 150;
+
+interface ThumbnailScrollView {
+  photos: (Partial<Photo> | undefined)[];
+  audioRecordings: any[];
+  observationId?: string;
+}
+
+export const ThumbnailScrollView: FC<ThumbnailScrollView> = props => {
```

Comment:
1. This component is still called `ThumbnailScrollView` but it's in a file called `Thumbnail/index.tsx`. Why did it change?
2. Given that this contains photo and audio thumbnails, maybe we give this a different name, like `MediaScrollView`?

Reply:
rename file